salt/beacons/__init__.py

```diff
@@ -125,16 +126,35 @@ def process(self, config, grains):
                         continue
 
                 # Fire the beacon!
-                raw = self.beacons[fun_str](b_config[mod])
-                for data in raw:
+                error = None
+                try:
+                    log.debug(self.beacons[fun_str])
```

Comment:
What does this print out? I ask, because my concern is it might not be useful to an end user.

Reply:
Left over from some debugging.  Removed.